cmd/internal/kube/util.go

```diff
@@ -130,3 +132,13 @@ func createPostmanSidecar(insightsProjectID string, addAPIKeyAsSecret bool) v1.C
 
 	return sidecar
 }
+
+func kubeCommandPreRun(cmd *cobra.Command, args []string) {
```

Comment:
Now that this is re-used, maybe add a comment explaining what it's doing (i.e., suppressing the normal INFO output.)